config/initializers/spree.rb

```diff
@@ -17,8 +17,8 @@
       .with_admin_ability_check(Spree::Slide)
```

Comment:
@sushil-truemark seems like you are skipping ability check

Reply:
I think you are recreating the function from line 14..and we should consolidate changes instead of just adding new configs..it will leave dead code and create confusion

Reply:
Unfortunately, I am not able to understand how we are recreating the function here and skipping the ability check. 
This applied way is one suggested by [spree documentation](https://dev-docs.spreecommerce.org/customization/admin-panel) and default menu items are creating using similar way. 

https://github.com/spree/spree_backend/blob/main/app/models/spree/admin/main_menu/default_configuration_builder.rb#L80



<img width="687" alt="image" src="https://github.com/spreehood/spree_slider/assets/133846865/e3a239d4-6754-4c9f-9c72-354b1b3499ea">

Reply:
Also, Deface is no longer recommended way.
https://dev-docs.spreecommerce.org/advanced/deface_overrides_tutorial
<img width="766" alt="image" src="https://github.com/spreehood/spree_slider/assets/133846865/e892e449-2a7a-407c-9df7-8c88a5701cbb">

Reply:
@sushil-truemark I am just reffering to line 17 I commented on..
it was checking ability previously in initializer